backend/grants/admin.py

```diff
@@ -278,6 +266,8 @@ class Meta:
             "travelling_from",
             "country_type",
             "applicant_message",
+            "applicant_reply_sent_at",
+            "applicant_reply_deadline",
```

Comment:
interesting that each grant has it is own deadline instead of being a global thing

Reply:
Because you can send the email separately, for example: if we receive more founds, we will send other emails with a different deadline of course, in 14 days from the date the email is sent. 
This is the deadline for the applicant to reply and confirm the grant, that is different to the `Waiting List Deadline` I added in the deadlines, that one is global and is for telling them when they will receive an update.